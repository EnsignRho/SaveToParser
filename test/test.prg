DECLARE INTEGER		process_save_to_file ;
						IN SaveToParser.dll ;
					STRING tcInputFile, ;
					STRING tcOutputFile

lcTest		= "Hello, world!"
lnValue		= 12345
lfValue		= 123.45678
lnBigNum	= 0 + (9999999999 - 9999999999)
ldDate		= DATE()
ltDateTime	= DATETIME()
SAVE TO test.mem

process_save_to_file("test.mem", "test.out")

lcText = FILETOSTR("test.out")
? lcText
