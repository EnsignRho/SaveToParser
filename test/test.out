(C:14)lcTest = Hello, world!
(N:10)lnValue = 12345
(N:16,5)lfValue = 123.45678
(N:13,2)lnBignum = 0.00
(D:8)ldDate = 07/10/2012
(T:8)ltDatetime = 07/10/2012 15:33:51.000
